packages/lit-dev-content/site/tutorials/content/wc-to-lit/11.md

```diff
@@ -58,9 +58,9 @@ What's changed?
 
 The `css` tag function allows you to define CSS tagged template literals, which enable math, templating, and other features under the hood.
 
-{% aside "info" %}
+{% aside "info" "no-header" %}
```

Comment:
Why does this one use a macro while the other ones use the tag directly?

Reply:
I meant to update all of them ahaha thanks for the catch